wix/Build.OpenJDK_generic.cmd

```diff
@@ -40,6 +40,11 @@ IF NOT DEFINED PRODUCT_UPDATE_INFO_LINK SET PRODUCT_UPDATE_INFO_LINK=https://ado
 IF NOT DEFINED WIX_HEAT_PATH SET WIX_HEAT_PATH=.\Resources\heat_dir\heat.exe
 IF NOT DEFINED WIX_VERSION SET WIX_VERSION=5.0.0
```

Comment:
Should that be 5.0.2?

Reply:
This change is part of a separate PR that I wanted to keep decoupled from this one. The link to that PR is: https://github.com/adoptium/installer/pull/1136